JASP-Engine/JASP/R/LSbinomialestimation.R

```diff
@@ -1065,7 +1077,8 @@ LSbinomialestimation   <- function(jaspResults, dataset, options, state = NULL){
           posterior      = temp_results[["distribution"]],
           posteriorEst   = temp_results[[options[["predictionTableEstimate"]]]],
           predictive     = temp_prediction[["distribution"]],
-          predictiveEst  = temp_prediction[[options[["predictionTableEstimate"]]]]
+          predictiveEst  = temp_prediction[[options[["predictionTableEstimate"]]]],
+          predictiveSD   = temp_prediction[["SD"]]
```

Comment:
It might be good to get rid of the snake cases now. Otherwise, we have to change them later with the qml upgrader, which is a pain.

Reply:
changed